service.go

```diff
@@ -19,6 +21,16 @@ type Service struct {
 	Logger      *logrus.Logger
 }
 
+func (svc *Service) GetUserID(c echo.Context) interface{} {
+	if svc.cfg.LNBackendType == LNDBackendType {
```

Comment:
hmm, do we really need that exception? 

I would actually add a login handler and just use the same session also for LND. 
Then also on umbrel the app is secured not only by the private network but a password (like terminal or other apps do it for example)

Reply:
I would keep things simple.

Reply:
password protection is out of scope for this PR.

Reply:
for me an `if` that has some exception is always an indication of complexity and not "simple".